start.py

```diff
@@ -513,7 +532,22 @@ def _apply_kerberos_fix_for_hue(cluster):
         'service krb5kdc restart',
         'service kadmin restart'
     ]
-    kdc_node.execute('; '.join(kdc_hue_commands))
+    kdc_node.execute('&& '.join(kdc_hue_commands))
+
+
+def _apply_kerberos_ticket_expiration_for_cm_principals(cluster, cm_kerberos_principals, kerberos_ticket_lifetime):
+    """ Apply kerberos ticket expiration for Kerberos principals needed by the services being managed by
```

Comment:
No space after opening triple quotes and be consistent with casing of "Kerberos."

Reply:
Done.